cypress/support/fragments/settings/dataImport/fileExtensions/newFileExtension.js

```diff
@@ -43,4 +43,11 @@ export default {
       dataTypeSelect.has({ filterValue: '' }),
     ]);
   },
+  creatNewFileExtension: (fileExtension) => {
+    cy.do([
```

Comment:
please create a function 'creatNewFileExtension' from those functions that already exist (fill,save)